rskj-core/src/main/java/co/rsk/peg/lockingcap/LockingCapStorageProvider.java

```diff
@@ -0,0 +1,12 @@
+package co.rsk.peg.lockingcap;
+
+import co.rsk.bitcoinj.core.Coin;
+
+public interface LockingCapStorageProvider {
+
+    Coin getLockingCap();
```

Comment:
Not sure, but if this can be null I would wrap the return value in an Optional.

Reply:
Done!